benchmarks/bench_isolation_forest.py

```diff
@@ -0,0 +1,101 @@
+"""
+==========================================
+IsolationForest benchmark
+==========================================
+
+A test of IsolationForest on classical anomaly detection datasets.
+
+"""
+print(__doc__)
+
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.ensemble import IsolationForest
+from sklearn.metrics import roc_curve, auc
+from sklearn.datasets import load_kddcup99, load_shuttle, fetch_covtype
+from sklearn.preprocessing import LabelBinarizer
+
+np.random.seed(1)
+
+
+datasets = ['http', 'smtp', 'SA', 'SF', 'shuttle', 'forestcover']
+
+for dat in datasets:
+    # loading and vectorization
+    print('loading data')
+    if dat in ['http', 'smtp', 'SA', 'SF']:
+        dataset = load_kddcup99(subset=dat, shuffle=True)
```

Comment:
Yes that is the problem with not using pandas.
That is the best solution I have found...